src/sbt-test/interplay/library-with-plugin/test

```diff
@@ -1,5 +1,6 @@
 # Setup local git repo
 $ exec git init
+$ exec git checkout -b main
```

Comment:
You can use `git init -b main`.

Reply:
Oh great! Travis' `git` is so old `git init -b` doesn't work. 🤦🏼‍♂️